src/main/java/cubicchunks/world/CubeWorldEntitySpawner.java

```diff
@@ -151,88 +130,64 @@ private int spawnCreatureTypeInAllChunks(EnumCreatureType mobType, ICubicWorldSe
             int blockY = blockpos.getY();
             int blockZ = blockpos.getZ();
 
-            int currentPackSize = 0;
```

Comment:
It seems like they are important part of the algorithm, but readding them will probably negate all the performance improvements you got with this.